### PR TITLE
fix arg name

### PR DESCRIPTION
**Pull request by erhoo82**
# What does this PR do ?

Fix arg name typo

# Changelog 
- change `tp_comm_disable_qkv` to `tp_comm_overlap_disable_qkv`

# Usage

# GitHub Actions CI

The Jenkins CI system has been replaced by GitHub Actions self-hosted runners.

The GitHub Actions CI will run automatically when the "Run CICD" label is added to the PR.
To re-run CI remove and add the label again.
To run CI on an untrusted fork, a NeMo user with write access must first click "Approve and run".

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: nemo/collections/nlp/modules/common/megatron/adapters/parallel_adapters.py [hunk 1]
```diff
@@ -257,7 +257,7 @@ def __init__(
 
             te_version = packaging.version.Version(version("transformer-engine"))
             if te_version >= packaging.version.Version("1.5.0dev") and (
-                not self.input_is_parallel and model_parallel_config.tp_comm_disable_qkv
+                not self.input_is_parallel and model_parallel_config.tp_comm_overlap_disable_qkv
```
[the remark this answers is not in the document]
Comment (erhoo82) [reply nemo/collections/nlp/modules/common/megatron/adapters/parallel_adapters.py:260]: I checked and the only place this arg is supposed to be used is MCORE.
However, we haven't merged the code lines that use this arg yet. So, I think it is totally fine.

[the remark this answers is not in the document]
Comment (ericharper) [reply nemo/collections/nlp/modules/common/megatron/adapters/parallel_adapters.py:260]: Should we set a default? Won't someone using an older version of mcore error here?

[the remark this answers is not in the document]
Comment (akoumpa) [reply nemo/collections/nlp/modules/common/megatron/adapters/parallel_adapters.py:260]: can we do a getattr to be on the safe side?

**Review by akoumpa — APPROVED** [review]
LGTM, thanks

**Review by akoumpa — APPROVED** [review]

**Pull request merged**